ring/partition_instance_lifecycler_test.go

```diff
@@ -0,0 +1,336 @@
+package ring
+
+import (
+	"context"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/go-kit/log"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/grafana/dskit/kv"
+	"github.com/grafana/dskit/kv/consul"
+	"github.com/grafana/dskit/services"
+)
+
+func TestPartitionInstanceLifecycler(t *testing.T) {
+	const eventuallyTick = 10 * time.Millisecond
+
+	ctx := context.Background()
+	logger := log.NewNopLogger()
+
+	t.Run("should wait the configured minimum number of owners before switching a pending partition to active", func(t *testing.T) {
```

Comment:
suggestion: "should wait" => "should wait for" (elsewhere too).

Reply:
Done in https://github.com/grafana/dskit/pull/483/commits/20dd7791eba5f795fee61484608e971905261df4